blog/electron-20.md

```diff
@@ -0,0 +1,71 @@
+authors:
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+slug: electron-20-0
+
+---
+
+Electron 20.0.0 has been released! It includes upgrades to Chromium `104`, V8 `10.4`, and Node.js `16.15.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 20.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### New Features
+
+* Added immersive dark mode on Windows. [#34549](https://github.com/electron/electron/pull/34549)
+* Added support for panel-like behavior. Window can float over full-screened apps. [#34665](https://github.com/electron/electron/pull/34665)
+* Updated Windows Control Overlay buttons to look and feel more native on Windows 11. [#34888](https://github.com/electron/electron/pull/34888)
+
+### Stack Changes
+
+* Chromium `104`
+    * [New in Chrome 104](https://developer.chrome.com/blog/new-in-chrome-104/)
+    * [New in Chrome 103](https://developer.chrome.com/blog/new-in-chrome-103/)
+    * [New in DevTools](https://developer.chrome.com/blog/new-in-devtools-104/)
+* Node.js `16.15.0`
+    * [Node 16.15.0 blog post](https://nodejs.org/en/blog/release/v16.15.0/)
+* V8 `10.4`
+
+## Breaking & API Changes
+
+Below are breaking changes introduced in Electron 20. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/latest/breaking-changes) page.
+
+### V8 Memory Cage Enabled
+
+Electron 20 enables [V8 sandboxed pointers](https://docs.google.com/document/d/1HSap8-J3HcrZvT7-5NsbYWcjfc0BVoops5TDHZNsnko/edit), following Chrome's [decision to do the same in Chrome 103](https://chromiumdash.appspot.com/commit/9a6a76bf13d3ca1c6788de193afc5513919dd0ed). This has some implications for native modules. This feature has performance and security benefits, but also places some new restrictions on native modules, e.g. use of ArrayBuffers that point to external ("off-heap") memory. Please see [this blog post](https://electronjs.org/blog/v8-memory-cage) for more information.
+
+### Default Changed: renderers without `nodeIntegration: true` are sandboxed by default
+
+Previously, renderers that specified a preload script defaulted to being unsandboxed. This meant that by default, preload scripts had access to Node.js. In Electron 20, this default has changed. Beginning in Electron 20, renderers will be sandboxed by default, unless `nodeIntegration: true` or `sandbox: false` is specified.
+
+If your preload scripts do not depend on Node, no action is needed. If your preload scripts do depend on Node, either refactor them to remove Node usage from the renderer, or explicitly specify `sandbox: false` for the relevant renderers.
+
```

Comment:
```suggestion

### Fixed: spontaneous crashing in nan native modules

In Electron 20, we changed two items related to native modules:
1. V8 headers now use `c++17` by default. This flag was added to electron-rebuild.
1. We fixed an issue where a missing include would cause spontaneous crashing in native modules that depended on nan.

For the most stability, we recommend using node-gyp >=8.4.0 and electron-rebuild >=3.2.9 when rebuilding native modules, particularly modules that depend on nan. See electron [#35160](https://github.com/electron/electron/pull/35160) and node-gyp [#2497](https://github.com/nodejs/node-gyp/pull/2497) for more information.

```

Reply:
This is a nice improvement! :+1: